src/kili/adapters/authentification.py

```diff
@@ -0,0 +1,37 @@
+"""Authentification functions that does not rely on the graphQL client."""
```

Comment:
it looks like a graphQL call, but it is not?

Reply:
it is a graphql call but without the client

Reply:
ok, clarified this morning